src/ngraph/runtime/interpreter/int_backend.cpp

```diff
@@ -134,35 +145,42 @@ bool runtime::interpreter::INTBackend::call(shared_ptr<Function> function,
         {
             continue;
         }
+        if (type_id == OP_TYPEID::Constant)
+        {
+            const op::Constant* c = static_cast<const op::Constant*>(op);
+            descriptor::Tensor* tv = op->get_output_tensor_ptr(0).get();
+            tensor_map.insert({tv, const_cast<void*>(c->get_data_ptr())});
+            continue;
+        }
         // get op inputs from map
-        vector<shared_ptr<runtime::HostTensor>> op_inputs;
+        vector<const void*> op_inputs;
         for (const descriptor::Input& input : op->get_inputs())
         {
             descriptor::Tensor* tv = input.get_output().get_tensor_ptr().get();
             op_inputs.push_back(tensor_map.at(tv));
         }
 
         // get op outputs from map or create
-        vector<shared_ptr<runtime::HostTensor>> op_outputs;
+        vector<void*> op_outputs;
+        vector<shared_ptr<runtime::HostTensor>> htv_outputs;
         for (size_t i = 0; i < op->get_output_size(); ++i)
         {
             descriptor::Tensor* tv = op->get_output_tensor_ptr(i).get();
-            shared_ptr<runtime::HostTensor> htv;
+            void* htv = nullptr;
```

Comment:
Might rename this variable since it is no longer a  `HostTensor` (but then there are vestigial `v`s throughout anyway, so maybe that's another PR).

Reply:
I figured since I am here already I will rename them, so I did.